app/controllers/admin/induction_coordinators_controller.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module Admin
+  class InductionCoordinatorsController < Admin::BaseController
+    skip_after_action :verify_authorized, only: :index
+    skip_after_action :verify_policy_scoped, except: :index
+    before_action :load_induction_coordinator, only: %i[edit update]
+
+    def index
+      @induction_coordinators = policy_scope(User).induction_coordinators
+    end
+
+    def edit
+      authorize @induction_coordinator
+    end
+
+    def update
+      authorize @induction_coordinator
+
+      if @induction_coordinator.update(permitted_attributes(@induction_coordinator))
+        redirect_to :admin_induction_coordinators, notice: "Changes saved successfully"
+      else
+        render :edit
+      end
+    end
+
+  private
+
+    def load_induction_coordinator
+      @induction_coordinator = User.find(params[:id])
+    end
```

Comment:
I guess we could authorise here if we cared about one line of repetition - I'll let you decide if we do :)

Reply:
I didn't want to make the edit action codeless and then have to do def edit; end which I find very ugly 😢 . I may do that in my next pr for admins editing other admins